context-menu-copy-link-with-types/background.js

```diff
@@ -1,8 +1,12 @@
+// Callback clears runtime.lastError to prevent logspam errors from the menu
```

Comment:
Please change it to:
"Callback reads runtime.lastError to prevent an unchecked error from being logged when the extension attempt to register the already-registered menu again. Menu registrations in event pages persist across extension restarts."